doc/helpers/StatusBar-extensions.md

```diff
@@ -0,0 +1,20 @@
+# StatusBar (attached properties)
+Provides two attached properties on `Page` to controls the visual of the status bar on mobile platforms.
+
+## Remarks
+The attached properties does nothing on platforms other than iOS and Android.
+For iOS, `UIViewControllerBasedStatusBarAppearance` should set to false in `info.plist`.
+
+## Properties
+Property|Type|Description
+-|-|-
+ForegroundTheme|StatusBarTheme|Sets the foreground color for the text and icons on the status bar. Possible values are: `None`, `Light`, `Dark`.
```

Comment:
It's difficult to understand the meaning of those values. What does the `None` means? (no status bar??) If I put `Dark`, does that means the foreground color will be dark or it will be an appropriate color for a dark background?

Reply:
None, it is supposed to unsubscribe the event. I have no found a way to reset the color / knowing the initial color, so it kind of just leave it on the last set color. 
Dark means dark. The behavior you described would be more like "Auto".